pkg/proxy/apis/config/fuzzer/fuzzer.go

```diff
@@ -42,6 +42,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			obj.FeatureGates = map[string]bool{c.RandString(): true}
 			obj.HealthzBindAddress = fmt.Sprintf("%d.%d.%d.%d:%d", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(65536))
 			obj.IPTables.MasqueradeBit = utilpointer.Int32Ptr(c.Int31())
+			obj.IPTables.LocalhostNodePorts = utilpointer.BoolPtr(true)
```

Comment:
you should actually fuzz this `c.RandBool()`

Reply:
Thanks for your point!